client/src/main/com/sinch/sdk/domains/verification/VerificationService.java

```diff
@@ -9,6 +9,17 @@
  */
 public interface VerificationService {
 
+  /**
+   * Use application secret in place of unified configuration for authentication (see Sinch
+   * dashboard for details)
+   *
+   * @param key see <a href="https://dashboard.sinch.com/voice/apps">dashboard</a>
+   * @param secret see <a href="https://dashboard.sinch.com/voice/apps">dashboard</a>
+   * @return service instance for project
+   * @since 1.0
+   */
+  VerificationService useSecrets(String key, String secret);
```

Comment:
What happens when user calls `verification()` without first calling this method?

Reply:
Currently, the server will return a 4011 / Authentication error because of the unified credentials will be used ... but not recognized by server